### ISSUE TITLE
Question: Is this being maintained anymore?

### ISSUE THREAD

**Issue by pizzapanther** [dcramer/django-devserver#127]
I see some recent activity but no release to pypi or bump in version? @egrubbs ?

**Comment by prokaktus** [comment]
It's hard to find suitable fork with good support. Maybe it will be better to add maintainers for this project, who have time to review PRs and issues? @egrubbs 

Because there exists some PRs with very important features as Python 3 support. But without tests it's always risky to merge anything. Maybe we should write some tests (for basic scenarios, at least) and then start merge important PRs.

**Comment by egrubbs** [comment]
@prokaktus, I don't have permission to add new maintainers. I became a maintainer by asking dcramer for access several months ago, but I have unfortunately not spent as much time on the project as I had hoped.

**Comment by prokaktus** [comment]
@egrubbs oh, thanks for info!

**Comment by jdejoode** [comment]
@dcramer do you want to add this project to jazzband?

**Comment by dcramer** [comment]
@jdejoode yeah I'm totally fine w/ that. I'm not sure its heavily used these days fwiw. We had gotten to a point where we were mostly using it for its improved runserver capabilities, but I think all of that might be in Django core now.

**Comment by jdejoode** [comment]
I think the line and memory profiler integration are still lacking in django core, or am I mistaking?

**Comment by dcramer** [comment]
They're not in core, you're correct. I've just found them to be less useful over the years.

**Comment by jdejoode** [comment]
@dcramer you are right. I tried pyinstrument and django-cprofile-middleware and they fit my usecase as well. So maybe it is not a good investment of your time. I'll be upfront, I do not have the bandwidth now to maintain it, so unless someone steps up... @egrubbs ?

Thanks for your answer.
